config/locales/proper.en.yml

```diff
@@ -53,6 +53,13 @@ en:
       accordion:
         header: "Browse Service Catalog"
       back_to_top: "Back to Top"
+      short_interaction: "Short Interaction"
+      submit: "Submit"
+      add: "Add"
+      continue: "Continue"
+      contact: "Contact Us"
+      help: "Help/FAQs"
```

Comment:
This value is pulled from `t(:proper)[:right_navigation][:faqs][:header]` in the application

https://github.com/sparc-request/sparc-request/blob/v3.5.0/app/helpers/service_requests_helper.rb